bots/image-refresh

```diff
@@ -133,17 +150,35 @@ def run(image, verbose=False, **kwargs):
 
     branch = task.branch(image, "images: Update {0} image".format(image), pathspec="bots/images", **kwargs)
     if branch:
-        pull = task.pull(branch, **kwargs)
 
-        # we want to validate all image refreshes with external projects
-        task.label(pull, ["test-external"])
+        pull = task.pull(branch, run_tests=False, **kwargs)
 
         # Trigger this pull request
         api = github.GitHub()
+        old_head = pull["head"]["sha"]
         head = api.get("pulls/{}".format(pull["number"]))["head"]["sha"]
+        retry = 3
+        while head == old_head and retry:
+            time.sleep(3)
+            head = api.get("pulls/{}".format(pull["number"]))["head"]["sha"]
+            retry -= 1
+            if not retry:
+                logging.warning("Failed to retrieve newer head after force pushing, which needed for triggering tests")
```

Comment:
I suggest raising a `RuntimeError`,  as otherwise the script will go on and work on obsolete data. It's also easier to spot in the logs.